newicons/score_hero.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+<svg width="48" height="48" version="1.1" viewBox="0 0 48 48" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
+<defs>
+<style>
+      .cls-1 {
+        fill: none;
+        stroke: #fff;
+        stroke-linecap: round;
+        stroke-linejoin: round;
+      }	
+</defs>
```

Comment:
```suggestion
<defs>
<style>
      .cls-1 {
        fill: none;
        stroke: #fff;
        stroke-linecap: round;
        stroke-linejoin: round;
      }	
</style>
</defs>
```
SVG file is not displayed because the `<style>` tag is not closed.

Reply:
Thank you for informing about the bug, I have fixed this and committed the changes.